gpu.Dockerfile

```diff
@@ -63,5 +63,6 @@ RUN pip install pycuda && \
     pip install pynvrtc && \
     /tmp/clean-layer.sh
 
+RUN conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults cudf=0.5 python=3.6
```

Comment:
Could you add the `/tmp/clean-layer.sh` (see above), and a unit test ([example](https://github.com/Kaggle/docker-python/pull/502/files))?

Reply:
Sure, we just released cudf 0.6 conda install. I will do unit test with the latest version. Thank you very much!

Reply:
@Philmod  Please see the new commits. Please note that two tests are ignored since they require external data. I tested them locally and both passed the tests.

Thank you very much!

Reply:
Sorry, I just realized that kaggle docker has been updated with cuda 10.0. I need to redo the test since it was done with cuda 9.2. Please don't review the current commit for now.

Thank you for understanding.

Reply:
I committed changes for cuda 10.0 install. I tested locally and all unit tests passed. Please see [comments](https://github.com/daxiongshu/docker-python/blob/rapids.ai/gpu.Dockerfile#L69) for details of running unit tests. Thank you very much.

Reply:
I first updated the `gpu.Dockerfile` with latest cuda 10.0 setup and then add rapids.ai install instructions. The file is not outdated anymore. Thank you for reviewing.

Reply:
@Philmod Please let me know if any further changes are requested. Thank you so much.

Reply:
I added unit tests in `tests` folder as the examples you gave. Thank you very much.